packages/Spinner/Spinner.jsx

```diff
@@ -154,7 +171,7 @@ class Spinner extends React.PureComponent {
               e.target.blur()
```

Comment:
I think this wrapper element cannot trigger `onFocus` because it isn't a focusable element itself. Set `tabindex={0}` so that it becomes focusable. Perhaps if this solution works, we may not need to recursively set attributes to all children.

Reply:
Focus events from any children will bubble up to this element, that being said. It doesn't really look like this is doing anything at all.